src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -50,8 +52,12 @@ const ItemUser = ({photo, username, followed, onPress, userid}) => {
         </View>
         <View style={styles.containerButton}>
           <Pressable style={styles.followAction(32, 32)}>
-            <Animated.View style={{position: 'absolute', opacity: followIconFadeAnimation}}>
-              <MemoIc_Checklist width={32} height={32} />
+            <Animated.View style={{position: 'absolute', top: 4, opacity: followIconFadeAnimation}}>
+              <IconChecklist
+                width={32}
+                height={32}
+                color={isAnon ? COLORS.anon_primary : COLORS.signed_primary}
+              />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `MemoIc_Checklist` to `IconChecklist` and the dynamic color assignment based on the `isAnon` prop is a good improvement for visual consistency and user experience. However, please make sure that the new icon component `IconChecklist` has been properly tested and renders correctly in different scenarios.



<!-- This is an auto-generated comment by OSS CodeRabbit -->